### PR TITLE
Remove `pretty-ms` dependency (move code into core) (patch)

### PR DESCRIPTION
**Pull request by KATT**
Discussion: #1626 

### What are the changes and their implications?

Remove `pretty-ms` dep. Most handlers should hopefully take less than a second so it shouldn't have much impact.

### CODE REVIEW

File: packages/core/src/invoke.ts [hunk 1]
```diff
@@ -60,8 +59,8 @@ export async function invokeWithMiddleware<TInput, TResult>(
 
       const result = await enhancedResolver(params, res.blitzCtx)
 
-      const duration = prettyMs(new Date().getTime() - startTime)
-      log.info(chalk.dim("Finished", "in", duration))
+      const duration = new Date().getTime() - startTime
+      log.info(chalk.dim(`Finished in ${duration}ms`))
```
Comment (flybayer) [inline packages/core/src/invoke.ts:63]: To remove pretty-ms, we need to replicate the full behavior. For example pretty-ms will convert to seconds if greater than 1000 ms.

**Review by flybayer — CHANGES_REQUESTED** [review]

File: packages/core/src/invoke.ts (hunk 1 above)
Comment (KATT) [reply packages/core/src/invoke.ts:63]: I'd say if you have handlers taking more than 1000 ms I'd say you have bigger problems

Comment (KATT) [reply packages/core/src/invoke.ts:63]: Seems like an irrelevant optimization for me, but I can do it

Comment (KATT) [reply packages/core/src/invoke.ts]: I reluctantly did it

**Review by flybayer — APPROVED** [review]
Thanks :)

**Pull request merged**